sdcm/cluster_aws.py

```diff
@@ -336,7 +336,7 @@ def _get_instances(self, dc_idx, az_idx=None):
         availability_zone = self.params.get('availability_zone').split(",")[az_idx] if az_idx is not None else None
         ec2 = ec2_client.EC2ClientWrapper(region_name=self.region_names[dc_idx],
                                           spot_max_price_percentage=self.params.get('spot_max_price'))
-        results = list_instances_aws(tags_dict={'TestId': test_id, 'NodeType': self.node_type},
+        results = list_instances_aws(tags_dict={'TestId': test_id, 'NodeType': self.node_type}, running=True,
```

Comment:
how much time it takes from provision to machine is running? If long enough we may have broken logic during test startup.

Reply:
it's 1-2 minutes, in provision step we are waiting for it, and communicating with the nodes, AFAIK

Reply:
or maybe just in azure, we are waiting for cloud-init (or that's in the test, not provision)

Reply:
seems like we are not waiting on provision side at all...
I'll need to run a few test to see it in action